deploy/cr.yaml

```diff
@@ -383,6 +383,26 @@ spec:
 #    initImage: perconalab/percona-server-mysql-operator:main
 #    containerSecurityContext:
 #      privileged: true
+#    pitr:
```

Comment:
I don't think we need to have it in CR. Or we need to leave a comment that it is just binlogServer, which is in the alfa state now.